### PR TITLE
Add contact model, route, controller, view

### PR DESCRIPTION
**Pull request by yana-glt**
Added contact model, router, view.
Added contactController with basic CRUD operations.

### CODE REVIEW

File: src/middleware/verifyToken.ts
Comment (FCosta20) [inline src/middleware/verifyToken.ts:1]: Last PR you added the try catch, why are you removing it now?

File: src/models/contact.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+export interface IContact {};
+export interface IContactModel extends IContact, Document {};
```
Comment (FCosta20) [inline src/models/contact.ts:4]: Why do you need IContactModel extending IContact?
Why not just use IContact?

Comment (yana-glt) [reply src/models/contact.ts:4]: I can. But as I remember from Java Laba it is best practice to use in DAO additional layer to change the database without a problem. I'm not planning to change it now, so I can remove it. But I considered it as best practice.

File: src/middleware/verifyToken.ts
Comment (yana-glt) [reply src/middleware/verifyToken.ts:1]: I realized that the error handler will now work on it

**Pull request merged**